compiler/rustc_middle/src/mir/coverage.rs

```diff
@@ -151,11 +151,11 @@ impl Debug for CoverageKind {
             BlockMarker { id } => write!(fmt, "BlockMarker({:?})", id.index()),
             CounterIncrement { id } => write!(fmt, "CounterIncrement({:?})", id.index()),
             ExpressionUsed { id } => write!(fmt, "ExpressionUsed({:?})", id.index()),
-            CondBitmapUpdate { id, value } => {
-                write!(fmt, "CondBitmapUpdate({:?}, {:?})", id.index(), value)
+            CondBitmapUpdate { id, value, decision_depth } => {
+                write!(fmt, "CondBitmapUpdate({:?}, {:?}, {:?})", id.index(), value, decision_depth)
             }
-            TestVectorBitmapUpdate { bitmap_idx } => {
-                write!(fmt, "TestVectorUpdate({:?})", bitmap_idx)
+            TestVectorBitmapUpdate { bitmap_idx, decision_depth } => {
+                write!(fmt, "TestVectorUpdate({:?}, {:?})", bitmap_idx, decision_depth)
```

Comment:
Nit: This could get a bit confusing when reading debug output; consider adding a `depth=` before printing the depth to make the output more obvious.